bio_hansel/main.py

```diff
@@ -59,6 +60,8 @@ def init_parser():
                         help='Subtyping summary output path (tab-delimited)')
     parser.add_argument('-O', '--output-tile-results',
                         help='Subtyping tile matching output path (tab-delimited)')
+    parser.add_argument('-OS', '--output-simple-summary',
```

Comment:
Let's change this to `parser.add_argument('-S', '--output-simple-summary',` unless you can think of a better single char short opt (`-e`?). We want to make the commandline options as intuitive and consistent as possible where short opts are dash followed by a char (e.g. `-x`) and longer opts are 2 dashes and some words delimited by dashes (e.g. `--show-x`)

Reply:
I'll change it to `-S`. Hopefully won't be confusing as `-s` corresponds to scheme.